entity/FedexEntities.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<entities xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://moqui.org/xsd/entity-definition-2.1.xsd">
+
+    <!--Default Authentication Entity for storing Credentials -->
+    <!--Entity to stored required credentials for Authentication-->
+    <entity entity-name="FedexTokenDetails" package="mantle.fedex">
+        <!--tokenId will always remain one-->
+        <field name="tokenId" type="id" is-pk="true"/>
+        <field name="grantType" type="text-long"/>
+        <field name="clientId" type="text-long"/>
```

Comment:
Add field description, clientId,  clientSecret and accountNumber should be encrypted